utils/src/main/java/org/apache/cloudstack/utils/bytescale/ByteScaleUtils.java

```diff
@@ -44,4 +45,14 @@ public static long mibToBytes(long mib) {
     public static long bytesToKib(long b) {
         return b / KiB;
     }
+
+    /**
+     * Converts bytes to mebibytes.
+     *
+     * @param b The value in bytes to convert to mebibytes.
+     * @return The parameter divided by 1024 * 1024 (1 MiB).
+     */
+    public static long bytesToMib(long b) {
```

Comment:
@GutoVeronezi 
b stands for bits
B stands for bytes

The method name should use KiB/MiB

Reply:
Thanks, @weizhouapache. I will create a PR to fix this typo.

Reply:
@weizhouapache, I created PR #6877 to fix the typos.